src/Products/PythonScripts/tests/testPythonScript.py

```diff
@@ -59,7 +58,6 @@ def readf(name):
 class VerifiedPythonScript(PythonScript):
 
     def _newfun(self, code):
-        verify(code)
         return PythonScript._newfun(self, code)
 
```

Comment:
Wouldn't it be better to rip out `VerifiedPythonScript` altogether, as its only purpose was to trigger `verfify()`?